File: nodejs/7 kyu/51.js
// https://www.codewars.com/kata/59656c69253c365e58000046/train/javascript

// Passed

function maxPossibleScore(obj, arr) {
  const scores = Object.keys(obj).map((question) =>
    arr.includes(question) ? obj[question] * 2 : obj[question]
  );
  const totalScore = scores.reduce((prev, curr) => prev + curr, 0);
  return totalScore;
}

output = maxPossibleScore({ a: 1, b: 2, c: 4 }, ["a", "c"]);
console.log(output);

File: python/7 kyu/51.py
# https://www.codewars.com/kata/559f44187fa851efad000087/train/python

# Passed

def seven_ate9(str_):
    result = str_.replace("797", "77")
    while result != result.replace("797", "77"):
        result = result.replace("797", "77")
        
    return result

output = seven_ate9("7979977732412979724979741")
print(output)